BParkingNano/python/electronsBPark_cff.py

```diff
@@ -87,7 +90,8 @@
         ptBiased = Var("userFloat('ptBiased')",float,doc="ptBiased from seed BDT 20 for pfEle"), 
         unBiased = Var("userFloat('unBiased')",float,doc="unBiased from seed BDT 20 for pfEle"), 
         mvaId = Var("userFloat('mvaId')",float,doc="MVA ID for low pT, 20 for pfEle"),
-        fBrem = Var("fbrem()",float,doc="brem fraction from the gsf fit",precision=8)
+        fBrem = Var("fbrem()",float,doc="brem fraction from the gsf fit",precision=8),
+        overlapPFindex = Var("userInt('overlapPFindex')",float,doc="index of overlapping pf in selected_pf_collection",precision=8),
```

Comment:
Given we sort the electron collection we have no way of retrieving the overlapping PF, maybe is better to just store a bool

Reply:
ahhh right! yes the bool should work. thanks